website/search/util.py

```diff
@@ -35,16 +35,18 @@ def build_query(qs='*', start=0, size=10, sort=None):
                 sort: 'desc'
             }
         ]
-
     return query
 
 
+# Match queryObject in search.js
 def build_query_string(qs):
     return {
         'query_string': {
             'default_field': '_all',
+            'fields': ['title^2', '_all', 'description^1.2'],
```

Comment:
Why did you choose these numbers? (May want to move them to a well-named constant)